client/src/components/Sub_Questions/ImageModal.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/no-array-index-key */
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Image } from 'cloudinary-react';
+
+export default function ImageModal({ display, photos, setPhotos }) {
+  const addImage = (files) => {
+    if (!files[0]) {
+      return null;
+    }
+    const formData = new FormData();
```

Comment:
oh this is a really cool way to make forms. Is there a reason you used this way instead of using a normal form element?

Reply:
formdata is more like a config thing with ajax request
https://developer.mozilla.org/en-US/docs/Web/API/FormData

Reply:
so yeah no I didn't use it for form element.

Reply:
i'll steal this